app/views/pair_requests/index.html.erb

```diff
@@ -9,7 +9,7 @@
         <div class="table-cell text-left p-1">Partner</div>
         <div class="table-cell text-left p-1">When</div>
         <div class="table-cell text-left p-1">Status</div>
-        <div class="table-cell text-left p-1">Actions</div>
+        <div class="text-left p-1 hidden md:table-cell">Actions</div>
```

Comment:
This eliminates the `hidden` display at `md` while the cells break are doing it at `sm`. This creates a funky look between these two breakpoints where the column is rendered but without the top label:
![image](https://github.com/agency-of-learning/PairApp/assets/88392688/d65b8fa0-ac33-4b99-aec4-194210991502)

Reply:
oops, that was a typo; will fix, thanks for the keen eye!